src/wallet/wallet.ts

```diff
@@ -157,6 +189,66 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     return this.authToken;
   }
 
+  /**
+   * Returns the balance for each token in tx, if the input/output belongs to this wallet
+   *
+   * This method is meant to keep compatibility with the old facade
+   *
+   * @param {Object} tx Transaction data with array of inputs and outputs
+   *
+   * @return {Object} Object with each token and it's balance in this tx for this wallet
+   **/
+  async getTxBalance(tx, optionsParam = {}) {
```

Comment:
Missing parameters types and return type

Reply:
Done in 9ffc52b249d1207bc156ec52fb4a486310447167